app/views/posts/index.html.erb

```diff
@@ -1,9 +1,11 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>User Posts</title>
-  </head>
-  <body>
-    <h1>Posts for a given User</h1>
-  </body>
-</html>
+<div>
+  <%= render partial: 'shared/user', locals: {user: @user} %>
+
+  <% @posts.each do |post| %>
+    <%= render partial: 'shared/post', locals: {post: post} %>
+    <%= render partial: 'shared/comment', locals: {post: post} %>
+  <% end %>
+
```

Comment:
- [ ] You are using a comment's partial, but the comments displayed on this post index page should display only the 5 recent comments using the `five_recent_comments`. The partial displays all the comments which is not right.